macros/edr/tests/test_utils/get_anomaly_query.sql

```diff
@@ -51,12 +54,25 @@
     {{- return(anomaly_query) -}}
 {% endmacro %}
 
-{%- macro is_score_anomalous_condition(sensitivity) -%}
-    {%- set spikes_only_metrics = ['freshness', 'event_freshness'] -%}
+{%- macro set_directional_anomaly(anomaly_direction, anomaly_score, sensitivity) -%}
+    {%- if anomaly_direction is not none -%}
+        {% if anomaly_direction == 'spike' %}
+            anomaly_score > {{ sensitivity }}
+        {% elif anomaly_direction == 'drop' %}
+            anomaly_score < {{ sensitivity * -1 }}
+        {% else %}
+            abs(anomaly_score) > {{ sensitivity }}
+        {% endif %}
+    {% else %}
+        abs(anomaly_score) > {{ sensitivity }}
+    {% endif %}
+{% endmacro %}
 
+{%- macro is_score_anomalous_condition(sensitivity, anomaly_direction) -%}
+    {%- set spikes_only_metrics = ['freshness', 'event_freshness'] -%}
```

Comment:
Should this list be here or in the global vars? Is it possible that users would want to change it?

Reply:
Actually I didn't add this part (it's in the PR because I changed the function)
I asked about it as well, this was placed specifically here because it's a special case, when checking freshness there's no reason to see anything other than a "spike"
I've been explained that a drop would never be interesting, if you think otherwise I can edit that part too @Maayan-s

Reply:
Got it, makes sense